packages/dmn-js-decision-table/src/features/decision-rules/editor.js

```diff
@@ -3,7 +3,7 @@ import DecisionRulesEditor from './DecisionRulesEditor';
 import Rules from './index';
 
 export default {
-  __depends__: [ DebounceInput, Rules ],
+  __depends__: [ DebounceInput, Rules, 'translate' ],
```

Comment:
Does this do anything? I think we must depend on a module. Depending on a `string` does not yield any meaningful result.